config/phobos.yml.example

```diff
@@ -95,3 +95,5 @@ listeners:
     # Apply this encoding to the message payload, if blank it uses the original encoding. This property accepts values
     # defined by the ruby Encoding class (https://ruby-doc.org/core-2.3.0/Encoding.html). Ex: UTF_8, ASCII_8BIT, etc
     force_encoding:
+    # Use this if custom backoff is required for a listener
+    backoff:
```

Comment:
I think you should add 
```
  min_ms: 500
  max_ms: 30000
```
Or some other values that overrides the default, to seal the deal with this example